src/Utilities/functions.jl

```diff
@@ -4,20 +4,20 @@ using Compat.Test
 """
     evalvariables(varval::Function, f::AbstractFunction)
 
-Returns the value of function `f` if each variable index `vi` is evaluate as `varval(vi)`.
+Returns the value of function `f` if each variable index `vi` is evaluated as `varval(vi)`.
 """
 function evalvariables end
 evalvariables(varval::Function, f::MOI.SingleVariable) = varval(f.variable)
 evalvariables(varval::Function, f::MOI.VectorOfVariables) = varval.(f.variables)
-evalvariables(varval::Function, f::MOI.ScalarAffineFunction) = sum(evalterm.(varval, f.terms)) + f.constant
+evalvariables(varval::Function, f::MOI.ScalarAffineFunction) = mapreduce(t->evalterm(varval, t), +, zero(f.constant), f.terms) + f.constant
```

Comment:
Style nit: use one-line functions only if they fit within one 80-character line. So in this case just do
```julia
function evalvariables(...)
    return ...
end
```